### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by emyrue**
- Added redux folder in src folder
- Create reducers
- Use action creators
- Use combineReducer function
- Create store

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Actions
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
+
```
Comment (Gambit142) [inline src/redux/books/books.js]: - [ ] Kindly create your actions using the _**duck pattern**_. It should look like this:

```
const ADD = 'bookstore/books/ADD'
```

#### Kindly implement this change. You can read this [article](https://github.com/erikras/ducks-modular-redux) to get a better understanding of duck patterns.

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// Action
+const CHECK = 'CHECK';
+
```
Comment (Gambit142) [inline src/redux/categories/categories.js]: - [ ] Kindly create your actions using the _**duck pattern**_. It should look like this:

```
const CHECK = 'bookstore/categories/CHECK'
```

#### Kindly implement this change. You can read this [article](https://github.com/erikras/ducks-modular-redux) to get a better understanding of duck patterns.

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @emyrue 👋, 

Good job so far!

## Highlights
- Installed `react-redux` ✔️
- Created action creators ✔️
- Created Reducers ✔️

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @emyrue , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**